fem/gslib.cpp

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#include "gslib.hpp"
+
+#ifdef MFEM_USE_GSLIB
+
+namespace mfem
+{
+
+#ifdef __GNUC__
+// Ignore warnings from the gslib header (GCC version)
+#pragma GCC diagnostic ignored "-Wunused-function"
+#endif // __GNUC__
+
+#include "gslib.h"
```

Comment:
Good job on including the library header only in this module  :+1:

Reply:
The include should probably be outside `namespace mfem` though...

Reply:
Is 32faaccca432f1701005599baf08cefbe3be5a96 better?

Reply:
Yes :+1: